src/ui/components/Assist/Assist.tsx

```diff
@@ -0,0 +1,284 @@
+import React, { useState } from "react";
+
+import Icon from "replay-next/components/Icon";
+import useLocalStorage from "replay-next/src/hooks/useLocalStorage";
+import { setViewMode } from "ui/actions/layout";
+import { setSelectedPrimaryPanel } from "ui/actions/layout";
+import Events from "ui/components/Events";
+import { shouldShowDevToolsNag } from "ui/components/Header/ViewToggle";
+import Confetti from "ui/components/shared//Confetti";
+import hooks from "ui/hooks";
+import { Nag } from "ui/hooks/users";
+import { useDismissNag } from "ui/hooks/users";
+import { UserInfo } from "ui/hooks/users";
+import { useTestInfo } from "ui/hooks/useTestInfo";
+import { getViewMode } from "ui/reducers/layout";
+import { useAppDispatch, useAppSelector } from "ui/setup/hooks";
+import { ViewMode } from "ui/state/layout";
+import {
+  shouldShowAddComment,
+  shouldShowAddCommentToLine,
+  shouldShowAddCommentToNetworkRequest,
+  shouldShowAddCommentToPrintStatement,
+  shouldShowAddUnicornBadge,
+  shouldShowBreakpointAdd,
+  shouldShowBreakpointEdit,
+  shouldShowConsoleNavigate,
+  shouldShowExploreSources,
+  shouldShowInspectComponent,
+  shouldShowInspectElement,
+  shouldShowJumpToCode,
+  shouldShowJumpToEvent,
+  shouldShowLaunchCommandPalette,
+  shouldShowQuickOpenFile,
+  shouldShowRecordReplay,
+  shouldShowSearchSourceText,
+  shouldShowTour,
+  shouldShowUseFocusMode,
+} from "ui/utils/onboarding";
+
+import styles from "./Assist.module.css";
+
+const useNagDismissal = () => {
+  const dismissNag = useDismissNag();
+  const dispatch = useAppDispatch();
+  const info = useTestInfo();
+
+  const dismissTourNag = () => {
+    const initialPrimaryPanel = "events";
+    dispatch(setSelectedPrimaryPanel(initialPrimaryPanel));
+    dismissNag(Nag.DISMISS_TOUR);
+  };
+
+  return { dismissTourNag };
+};
+
+const Assist: React.FC = () => {
+  const [selectedItem, setSelectedItem] = useState(1);
```

Comment:
If this is a 0-based value, let's call it "index". If it's a 1-based value, let's call it "step" or "number" or something. ( Otherwise it's likely to cause off-by-one error to someone less familiar with the code later.)

Also– why does it default to 1 when there is a 0? Is this a bug?